.github/workflows/_meta-publish.yaml

```diff
@@ -0,0 +1,66 @@
+﻿on:
+  workflow_call:
+    inputs:
+      jellyfin_version:
+        required: true
+        type: string
+      server_url:
+        required: true
+        type: string
+
+jobs:
+  publish:
+    runs-on: windows-latest
+    env:
+      DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Install NSIS
+        run: |
+          Invoke-WebRequest 'https://get.scoop.sh' -OutFile 'scoop-install.ps1'
+          .\scoop-install.ps1 -RunAsAdmin
+          scoop bucket add extras
+          scoop install nsis
```

Comment:
Not that it hurts to do this, but NSIS 3.08 (the current stable) is already pre-installed on the GHA Windows image: https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md#tools

Reply:
Well shoot that would have saved me a few hours...